appscale/tools/agents/azure_agent.py

```diff
@@ -1193,3 +1214,92 @@ def does_resource_group_exist(self, resource_group_name, resource_client):
     if resource_group_name in resource_group_names:
       return True
     return False
+
+  def create_image(self, instance_id, image_name, parameters, dest_container):
```

Comment:
@cdonati So I agree that the user shouldn't use this, but (as far as I can see) there isn't a way for the user to call this method and a large majority of it seems tied to the azure agent in my opinion.